tests/nested/manual/refresh-from-old-base/task.yaml

```diff
@@ -0,0 +1,67 @@
+summary: Test refreshing from old snapd and base prior to required symlink
+
+details: |
+    Base snaps used to not initialize /snap/snapd/current symlink to seed.
+    However the question is what happen when starting with an old image,
+    and updating snapd before the base. This makes sure that this initial
+    symlink is not involved with updates.
+
+systems:
+  - ubuntu-22.04-64
+
+environment:
+  NESTED_BUILD_SNAPD_FROM_CURRENT: false
+  NESTED_REPACK_BASE_SNAP: false
+  NESTED_FORCE_SNAKEOIL_KEYS: true
+
+prepare: |
+    snap download --channel="latest/${BASE_CHANNEL}" core22 --basename=core22-new
+
+    # shellcheck source=tests/lib/prepare.sh
+    . "$TESTSLIB"/prepare.sh
+    [ -d new-snap/ ] || mkdir -p new-snap/
```

Comment:
Either the test or the -p argument is enough. Together they are redundant.

Reply:
I have changed it to clean up before in case we run several times the test and a previous run failed.